config/pkg/pldconf/sequencermgr.go

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright © 2025 Kaleido, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package pldconf
+
+import (
+	"time"
+
+	"github.com/LF-Decentralized-Trust-labs/paladin/config/pkg/confutil"
+)
+
+type SequencerConfig struct {
+	AssembleTimeout               *string           `json:"assembleTimeout"`
+	RequestTimeout                *string           `json:"requestTimeout"`
+	BlockHeightTolerance          *uint64           `json:"blockHeightTolerance"`
+	BlockRange                    *uint64           `json:"blockRange"`
+	ClosingGracePeriod            *int              `json:"closingGracePeriod"`
+	MaxInflightTransactions       *int              `json:"maxInflightTransactions"`
+	MaxDispatchAhead              *int              `json:"maxDispatchAhead"`
```

Comment:
Definitely not an immediate thing, but I wonder over time if some of these need to negotiated smart contract scoped config - rather than independent node config. It's the same kind of balance as something like max gas per block in the underlying ledger consensus.